src/VaultSharp/VaultSharp.csproj

```diff
@@ -35,6 +35,12 @@
     <WarningLevel>4</WarningLevel>
     <DocumentationFile>bin\Release\VaultSharp.XML</DocumentationFile>
   </PropertyGroup>
+  <PropertyGroup>
+    <SignAssembly>true</SignAssembly>
+  </PropertyGroup>
+  <PropertyGroup>
+    <AssemblyOriginatorKeyFile>VaultSharp.snk</AssemblyOriginatorKeyFile>
```

Comment:
cool. i agree that digitally signing the assembly is not required.

but to generate the strong named assembly for NuGet I still need to procure/purchase a cert.